telegram/_bot.py

```diff
@@ -906,7 +906,9 @@ async def send_photo(
         """Use this method to send photos.
 
         .. seealso:: :meth:`telegram.Message.reply_photo`, :meth:`telegram.Chat.send_photo`,
-            :meth:`telegram.User.send_photo`
+            :meth:`telegram.User.send_photo`,
+            `Working with Files and Media <https://github.com/\
+            python-telegram-bot/python-telegram-bot/wiki/Working-with-Files-and-Media>`_
```

Comment:
```suggestion
            |files_and_media_wiki|
```

Maybe we can use a subsitution for this?

Reply:
Absolutely, but maybe in a separate PR then? I mean, there are lots of similar links that could be in substitutions.

Reply:
I'd prefer to use a substitution here right away instead of changing it later. There are many links/docstrings/notes/… that can benefit of substitutions, so IMO it's more sane to convert those bit by bit rather than trying to change everything at once …

Reply:
OK! A regular `|substitution|` won't work with hyperlinks, so I tried using `extlinks` in commit 8e5ae0408d914fbb7709f9550386e40fdbf4cd95

I would also love to find a way of replacing hyphens with whitespaces automatically. Then it would say
```
:wiki:`Working-with-Files-and-Media`
```

instead of the custom link text:

```
:wiki:`Working with Files and Media <Working-with-Files-and-Media>`
```

But it doesn't seem possible. I tried putting something like `"%s".replace(...)` [here](https://github.com/python-telegram-bot/python-telegram-bot/pull/3423/commits/8e5ae0408d914fbb7709f9550386e40fdbf4cd95#diff-008dcb3426febd767787b1521f1fe33086313b927ea37eaab86df5fa88a51698R54), but it doesn't do the trick.